app/src/main/java/com/xmartlabs/daydreaming/ui/OnboardingFragment.java

```diff
@@ -0,0 +1,88 @@
+package com.xmartlabs.daydreaming.ui;
+
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.annotation.LayoutRes;
+import android.support.annotation.Nullable;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+import android.widget.VideoView;
+
+import com.hannesdorfmann.fragmentargs.annotation.FragmentWithArgs;
+import com.xmartlabs.daydreaming.R;
+
+import java.util.Locale;
+
+import butterknife.BindView;
+import butterknife.OnClick;
+
+/**
+ * Created by scasas on 4/5/17.
+ */
+@FragmentWithArgs
+public class OnboardingFragment extends BaseFragment {
+  @BindView(R.id.background_video_view)
+  VideoView backgroundVideoView;
+  @BindView(R.id.daydreaming_image_view)
+  ImageView daydreamingImageView;
+  @BindView(R.id.splash_view)
+  RelativeLayout splashView;
+  @BindView(R.id.square_shape_view)
+  View squareShapeView;
+  @BindView(R.id.start_dreaming_view)
+  TextView startDreamingView;
+
+  @LayoutRes
+  @Override
+  protected int getLayoutResId() {
+    return R.layout.onboarding_fragment;
+  }
+
+  @Override
+  public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
+    super.onViewCreated(view, savedInstanceState);
+    initializeAlphaValues();
+    setupBackgroundVideo();
+    setupAnimation();
+  }
+
+  @OnClick(R.id.start_dreaming_view)
+  void onClickedStartDayDreaming() {
+    Intent intent = Henson.with(getContext())
+        .gotoDashboardActivity()
+        .build()
+        .setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
+    getContext().startActivity(intent);
+  }
+
+  private void setupBackgroundVideo() {
+    String uriString = String.format(Locale.getDefault(), "android.resource://%s/%d", getActivity().getPackageName(), R.raw.daydreamingvideo);
```

Comment:
Rename daydreamingvideo to a more descriptive name. In addition, do snake case for resources. day_dreaming_video for example